java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1752,6 +1752,40 @@ public ColumnVector substring(ColumnVector start, ColumnVector end) {
     }
   }
 
+  /**
+   * Returns a new strings column where target string within each string is removed.
+   *
+   * @param target String to search for within each string.
+   * @return A new java column vector where target string is removed.
+   */
+  public ColumnVector stringReplace(Scalar target) {
+    Scalar replace = Scalar.fromString("");
+    return stringReplace(target, replace);
+  }
+
+  /**
+   * Returns a new strings column where target string within each string is replaced with the specified
+   * replacement string.
+   * Specifing an empty string for replace will essentially remove the target
+   * string if found in each string. Null string entries will return null output string entries.
+   *
```

Comment:
Does this replace the first instance found, or all instances found?  If it is first is it searching left to right?

What about the case that java decided to document in their replace.

> The replacement proceeds from the beginning of the string to the end, for example, replacing "aa" with "b" in the string "aaa" will result in "ba" rather than "ab".

Does this also match that?

Reply:
It replaces all instances found.  Yes, it also does the replacement from the beginning of the string to the end. I will add this example in unit tests and document here as well. Thanks!